<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QGridLayout" name="graphics">
      <item row="1" column="1">
       <widget class="QLabel" name="third_cam">
        <property name="text">
         <string>third_cam</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="radar">
        <property name="text">
         <string>Radar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="unused">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dash_cam">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>dash_cam</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="spawnVehicle">
        <property name="text">
         <string>Spawn Actor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="spawnNPC">
        <property name="text">
         <string>Spawn NPC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Select Mode</string>
        </property>
        <widget class="QRadioButton" name="manualControlButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>195</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Manual Mode</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="assistantModeButton">
         <property name="geometry">
          <rect>
           <x>211</x>
           <y>30</y>
           <width>194</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Adaptive Assistant Mode</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="autoModeButton">
         <property name="geometry">
          <rect>
           <x>411</x>
           <y>30</y>
           <width>195</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Automatic Mode</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_info">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_log">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRecord">
        <property name="text">
         <string>Record</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
